nightingale_dispatcher/src/nightingale_dispatcher/send_interface_request_task.py

```diff
@@ -2,12 +2,35 @@
 
 import rospy
 from nightingale_dispatcher.task import Task
+from nightingale_ros_bridge import BridgeConfig, UserInputs
+from nightingale_msgs.srv import InterfaceCall
 
 
 class SendInterfaceRequestTask(Task):
     def __init__(self):
         pass
 
     def execute(self, robot_state):
-        user_input = self.interface_comms_request()
-        return user_input
+        rospy.wait_for_service(BridgeConfig.UPDATE_UI_SERVICE, InterfaceCall)
+        try:
+            # spin up proxy when needed
+            self.interface_comms_proxy = rospy.ServiceProxy(
+                BridgeConfig.UPDATE_UI_SERVICE, InterfaceCall
+            )
+            user_input = self.interface_comms_proxy(robot_state)
+            # convert input to return code
+            status = Task.SUCCESS
```

Comment:
looks a bit redundant but should be useful to deal with any other return data if we decide to send info to tablet

Reply:
I'm not sure this is the best way to do this. Is to use a more generic Task constant that is general enough to apply to all tasks, then map the UserInput constants to the Task constants? If not, then it's better to just use the Task constants

Reply:
I'm not sure if it's possible to have a more generic task constant since the value returned has to  represent 3 or more different things. I'm thinking we either return the Task constant and also another value for the user input. Or we can have the mission planner own the constants but I'm not sure if I like that since the mission planner is just reading the values in and the ros bridge is the one who owns the communication between the interface